scripts/run_scrnaseq.R

```diff
@@ -0,0 +1,58 @@
+paramsList = list()
```

Comment:
Yes and no. Can you have a look at Andreas script run_scrnaseq_hto.R? It is written in a way that you can start it directly from command-line. That is in our case, we start the HTO script through a our Cell Ranger snakemake pipeline. 

Would you mind adapting your script to the style of the hto script? That would be super helpful.